documentation/ntfs/ntfs_0.5.md

```diff
@@ -0,0 +1,668 @@
+        Indiquer un commentaire (optionnel) de type TAD via les fichiers comments.txt et comment_links.txt
+
+### geometries.txt (optionnel)
+Ce fichier contient la représentation spatiale d'une géométrie (pour des lignes, parcours et/ou circulations). Chaque ligne du fichier représente une géométrie complète de l'objet.
+
+Colonne | Type | Contrainte | Commentaire
+--- | --- | --- | ---
+geometry_id | chaine | Requis | Identifiant de la géométrie. 
+geometry_wkt | géométrie | Requis | Représentation spatiale de la géométrie selon le standard http://fr.wikipedia.org/wiki/Well-known_text.
+
+    Les lignes et parcours peuvent être des LINESTRING ou des MULTILINESTRING.
+    Les circulations ne peuvent être que des LINESTRING. Si une MULTILINESTRING est spécifiée, seule la première LINESTRING sera utilisée.
+    Les points d'arrêts sont des POINT.
+    Les zones d'arrêt peuvent être des POINT, POLYGON ou MULTIPOLYGON.
+    Les zones géographiques et communes peuvent être des POLYGON ou MULTIPOLYGON.
+
+    Seules les types de géométries spécifiées sont retenues, les autres types géométries sont ignorées.
+    Le format du fichier est volontairement simple, une évolution pourra être envisagée si le besoin est rencontré.
+
+### object_properties.txt (optionnel)
+Ce fichier contient la description des propriétés complémentaires sur les différents objets du référentiel.
+Ces propriétés sont sous forme de liste de clés / valeurs qui doivent être standardisées par processus.
+
+Colonne | Type | Contrainte | Commentaire
+--- | --- | --- | ---
+object_type | chaine | Requis | Type d'objet sur lequel la propriété porte (line, route, trip, stop_area, stop_point)
+object_id | chaine | Requis | Identifiant de l'objet sur lequel la propriété porte
+object_property_name | chaine | Requis | Nom de la propriété complémentaire (texte libre)
+object_property_value | chaine | Requis | Valeur de la propriété complémentaire (texte libre)
+
+### object_codes.txt (optionnel)
+Ce fichier contient la liste des codes d'identification complémentaires dans les systèmes externes des différents objets du référentiel.
+Ces propriétés sont sous forme de liste de clés / valeurs qui doivent être standardisées par processus.
+
+Colonne | Type | Contrainte | Commentaire
+--- | --- | --- | ---
+object_type | chaine | Requis | Type d'objet sur lequel la propriété porte (network, line, route, trip, stop_area, stop_point)
+object_id | chaine | Requis | Identifiant de l'objet sur lequel la propriété porte
+object_system | chaine | Requis | Nom du système d'identification de l'objet  (texte libre). Par exemple : "Timeo" ou "UIC" pour les arrêts, "Reflex" pour les lignes.
+object_code | chaine | Requis | Code d'identification de l'objet dans le système considéré.
+
+Kisio Digital fournit dans ce fichier :
+* les identifiants des objets dans l'ancien système Navitia pour les objets "network", "line", "route", "trip", "stop" avec pour object_system la chaine ** "navitia1" **.
```

Comment:
The bold is missing here, maybe because of the spaces, it's good on the next line